src/main/java/uk/co/ramp/AppConfig.java

```diff
@@ -33,16 +35,34 @@
 
   private static final Logger LOGGER = LogManager.getLogger(AppConfig.class);
   private static final String DEFAULT_INPUT_FOLDER = "input";
+  private static final String DEFAULT_OUTPUT_FOLDER = "output";
   private static final String INPUT_FILE_LOCATION = "input/inputLocations.json";
 
   private final String seed;
   private final String overrideInputFolderLocation;
+  private final String overrideOutputFolderLocation;
 
   AppConfig(
       @Value("${seed:#{null}}") String seed,
-      @Value("${overrideInputFolderLocation:#{null}}") String overrideInputFolderLocation) {
+      @Value("${overrideInputFolderLocation:#{null}}") String overrideInputFolderLocation,
+      @Value("${overrideOutputFolderLocation:#{null}}") String overrideOutputFolderLocation) {
     this.seed = seed;
     this.overrideInputFolderLocation = overrideInputFolderLocation;
+    this.overrideOutputFolderLocation = overrideOutputFolderLocation;
+  }
+
+  @Bean
+  public File outputFolder() {
```

Comment:
I think with Beans, the scope is singleton, so you'd only be able to ever have one `File` instance which could be injected in multiple places. This would become problematic if we want to define another Bean with a return type of `File`. As `File` is a very generic class, I think it would be better to make the return type more specific. Perhaps a wrapper class called `OutputFiles` would do?